test/integration/shoots/logging/logging.go

```diff
@@ -0,0 +1,279 @@
+
+	framework.CBeforeEach(func(ctx context.Context) {
+		checkRequiredResources(ctx, f.SeedClient)
+		//Get the Fluent-Bit DaemonSet from the seed
+		err := f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBit)
+		framework.ExpectNoError(err)
+		//Get the Fluent-Bit ConfigMap from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitConfigMapName}, fluentBitConfMap)
+		framework.ExpectNoError(err)
+		//Get the Fluent-Bit Service from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBitService)
+		framework.ExpectNoError(err)
+		//Get the Fluent-Bit ClusterRole from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitClusterRoleName}, fluentBitClusterRole)
+		framework.ExpectNoError(err)
+		//Get the Fluent-Bit Rolebinding from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitClusterRoleName}, fluentBitClusterRoleBinding)
+		framework.ExpectNoError(err)
+		//Get the Fluent-Bit ServiceAccount from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: fluentBitName}, fluentBitServiceAccount)
+		framework.ExpectNoError(err)
+		//Get the cluster CRD from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: "", Name: "clusters.extensions.gardener.cloud"}, clusterCRD)
+		framework.ExpectNoError(err)
+		//Get the Loki StS from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiSts)
+		framework.ExpectNoError(err)
+		//Get the Loki ServiceAccount from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiServiceAccount)
+		framework.ExpectNoError(err)
+		//Get the Loki Service from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiName}, lokiService)
+		framework.ExpectNoError(err)
+		//Get the Loki ConfigMap from the seed
+		err = f.SeedClient.Client().Get(ctx, types.NamespacedName{Namespace: v1beta1constants.GardenNamespace, Name: lokiConfigMapName}, lokiConfMap)
+		framework.ExpectNoError(err)
+	}, initializationTimeout)
+
+	f.Beta().Serial().CIt("should get container logs from loki for all namespaces", func(ctx context.Context) {
+		ginkgo.By("Deploy the garden Namespace")
+		gardenNamespace = &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: v1beta1constants.GardenNamespace,
+			},
+		}
+		err := create(ctx, f.ShootClient.Client(), gardenNamespace)
+		framework.ExpectNoError(err)
```

Comment:
What do you think about?
```suggestion
		framework.ExpectNoError(create(ctx, f.ShootClient.Client(), gardenNamespace))
```